examples/blog-engine/src/BasicAuth.elm

```diff
@@ -25,31 +25,26 @@ parseAuth base64Auth =
 
 
 withBasicAuth :
-    Parser requestData
```

Comment:
The `Parser` type was giving compile errors. I’m a bit unsure what the purpose of this module is since I think it’s unused but I fixed the compile errors by copying the `withBasicAuth` function found here https://github.com/dillonkearns/elm-pages/blob/b0138218ba16ea2fe7aedbf3d8d003386155eb32/examples/pokedex/app/Route/BasicAuth.elm#L102-L123

Reply:
I probably copied it over by mistake, I would be in favor of removing this file 👍